Bottom: 2ee44df5cb00f5bb4fa6f0c8d9b906d3d34ade22
Top:    07785a4f95510b97840ca608b834adde960b5045
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-21 00:50:59 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 61eacba..8f698f4 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -283,6 +283,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
